Program.cs

```diff
@@ -233,6 +233,47 @@ static async Task Main(string[] _)
 
             await ReadyEvent.OnStartup(discord);
             
+            // Migration from joinwatch to user notes
+            var joinWatchedUsersList = await Program.db.ListRangeAsync("joinWatchedUsers");
+            var joinWatchNotesList = await Program.db.HashGetAllAsync("joinWatchedUsersNotes");
+            int successfulMigrations = 0;
+            int numJoinWatches = joinWatchedUsersList.Length;
+            foreach (var user in joinWatchedUsersList)
+            {
+                // Get text for note; use joinwatch context if available, or "N/A; created from joinwatch without context" otherwise
+                string noteText;
+                if (joinWatchNotesList.FirstOrDefault(x => x.Name == user) == default)
+                    noteText = "N/A; created from joinwatch without context";
```

Comment:
nit
```suggestion
                // Get text for note; use joinwatch context if available, or "N/A; imported from joinwatch without context" otherwise
                string noteText;
                if (joinWatchNotesList.FirstOrDefault(x => x.Name == user) == default)
                    noteText = "N/A; imported from joinwatch without context";
```